src/driver/test_point_aggregator.py

```diff
@@ -124,7 +124,21 @@ def aggregate(
     Returns:
       A single row DataFrame representing the values of the statistics listed in keys.
     """
-    stats = {}
+    stats = {"model_succeeded": [1], "model_exception": [""]}
     for key in AGGREGATORS:
         stats[key] = [AGGREGATORS[key](true_reach, simulated_reach)]
     return pd.DataFrame(data=stats)
+
+
+def aggregate_on_failure(inst: Exception) -> pd.DataFrame:
+    """Returns a DataFrame of the same shape as aggregate but for case of failure.
+
+    Args:
+      inst:  The exception instance that was generated in the modeling attempt.
+    Returns:
+      A single row DataFrame representing the values of the statistics listed in keys.
```

Comment:
perhaps say "A single row DataFrame of NAs with columns being the statistics listed in keys".

Reply:
That is also better.